com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/GameObjectBuilder.java

```diff
@@ -176,6 +179,7 @@ public void build(Task<Void> task) throws CompileExceptionError,
                     .setId(ec.getId())
                     .setPosition(ec.getPosition())
                     .setRotation(ec.getRotation())
+                    .setScale(ec.hasScale() ? ec.getScale() : defaultScale)
```

Comment:
It is not possible to set a default value for a v3 in the .proto file so we need to detect this here instead. If the component doesn't have a scale we set a default value of v3(1.0). If we do not set a default here the scale will be v3(0.0).